go/consensus/tendermint/abci/mux.go

```diff
@@ -320,12 +368,12 @@ func (mux *abciMux) Query(req types.RequestQuery) types.ResponseQuery {
 			"req", req,
 		)
 		return types.ResponseQuery{
-			Code: api.CodeOK.ToInt(),
+			Code: types.CodeTypeOK,
 		}
 	}
 
 	return types.ResponseQuery{
-		Code: api.CodeInvalidQuery.ToInt(),
+		Code: types.CodeTypeOK,
```

Comment:
not sure if it matters really, but the code here used to be `api.CodeInvalidQuery`

Reply:
Yeah this is on purpose as we don't use (and will not use) the Tendermint query interface (as we have our own that can avoid all the re-serialization/deserialization). Could also add a random fail code instead.

Reply:
I'll add a comment to make this more clear.

Reply:
Maybe we should just remove `Query`.

Reply:
Removed.